addons/metallb/20_metallb-config.yaml

```diff
@@ -0,0 +1,24 @@
+{{- range $ipamPool, $allocation := .Cluster.Network.IPAMAllocations }}
```

Comment:
You are missing copyright header in this one.

Reply:
oh my bad